yarn-project/barretenberg.js/src/crypto/pedersen/pedersen.test.ts

```diff
@@ -17,11 +17,11 @@ describe('pedersen', () => {
   });
 
   it('hasher_consistency_and_benchmark', () => {
-    const start1 = new Date().getTime();
+    // const start1 = new Date().getTime();
     const result = pedersenGetHashTree(barretenbergWasm, values);
-    const end1 = new Date().getTime() - start1;
+    // const end1 = new Date().getTime() - start1;
 
-    console.log(`Single hasher: ~${end1 / values.length}ms / value`);
```

Comment:
left comments

Reply:
Hello, I think here it's ok to leave the `console.log` statements commented out because it's just for when people run it locally + the whole test is a messy WIP.